plugins/table.md

```diff
@@ -696,12 +696,13 @@ tinymce.activeEditor.plugins.table.insertTable(2, 3);
 
 ## Events
 
-| Name | Description |
-|------| ----------- |
-| newrow | Fired when a row is created |
-| newcell | Fired when a cell is created |
-| ObjectResizeStart | Fired when a resize action is started on a table, row, column or cell using the resize bars |
-| ObjectResized | Fired when a resize action is finished on a table, row, column or cell using the resize bars |
+| Name | Data | Description |
+|------| ---- | ----------- |
+| newrow | N/A | Fired when a row is created |
+| newcell | N/A | Fired when a cell is created |
+| ObjectResizeStart | N/A | Fired when a resize action is started on a table, row, column or cell using the resize bars |
+| ObjectResized | N/A | Fired when a resize action is finished on a table, row, column or cell using the resize bars |
+| tablemodified | `structure: bool; style: bool` | Fired whenever a table is modified in any way. The `style` and `structure` parameters describe whether the table style, structure, or both were modified. |
```

Comment:
```suggestion
| TableModified | `{ structure?: boolean; style?: boolean }` | Fired whenever style or structural changes are made to a table. The `style` and `structure` parameters describe whether the table style, structure, or both were modified. |
```

Reply:
Note: I made `structure` and `style` optional in the above, as we aren't always passing them through atm.

Reply:
I added "The **optional** `style` and `structure` parameters..." too